### PR TITLE
Minor style and English grammar suggestions

### PR DESCRIPTION
**Pull request by jhwohlgemuth**
Friendly peer review of the first section of the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,16 @@ A collection of common interactive command line user interfaces.
 
 <img align="right" alt="Inquirer Logo" src="/assets/inquirer_readme.png" title="Inquirer.js"/>
 
-We strive at providing easily embeddable and beautiful command line interface for Node.js; some hope in becoming the CLI Xanadu.
+**```Inquirer.js```** strives to be an easily embeddable and beautiful command line interface for [Node.js](https://nodejs.org/).
```
Comment (SBoudrias) [inline README.md]: Just need single backtick for inline code highlight.

Comment (SBoudrias) [inline README.md]: Also, please keep the Xanadu reference.

File: README.md [hunk 2]
```diff
@@ -292,4 +297,4 @@ to @vaxilart) or just add your name to [the wiki](https://github.com/SBoudrias/I
 ## License
 
 Copyright (c) 2012 Simon Boudrias (twitter: @vaxilart)  
-Licensed under the MIT license.
\ No newline at end of file
+Licensed under the MIT license.
```
Comment (SBoudrias) [inline README.md]: Note sure what changes on this line, but keep a line return on last line.

**Comment by jhwohlgemuth** [comment]
I wanted to include a link to what "Xanadu" is... but I was not sure what, in fact, it is... My best guess is "an idealized place of great or idyllic magnificence and beauty."?

**Comment by SBoudrias** [comment]
It's the castle of Citizen Kane :)

**Comment by jhwohlgemuth** [comment]
Castle of Citizen Kane...I of course...knew that... hehe...
I will update my PR with a link to the wiki article for Xanadu

**Comment by SBoudrias** [comment]
Alright, LGTM.

Mind squashing these commits? I'll merge once that's done.

**Comment by jhwohlgemuth** [comment]
Commits have been sufficiently squashed

**Comment by SBoudrias** [comment]
Sorry, doesn't look like it worked. You now have 5 commits on this CR.

**Comment by jhwohlgemuth** [comment]
I am making this way too hard... Perhaps you should just close this PR (because of my inability to squash commits).  I have created a second PR with a single commit.

File: README.md (hunk 2 above)
Comment (jhwohlgemuth) [inline README.md]: ...and I am not sure why I keep inadvertently removing the line return.

**Pull request closed without merging**